xml/System.Xaml/XamlXmlReaderSettings.xml

```diff
@@ -175,7 +175,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The XML whitespace concepts are not robust enough to apply to XAML, and are typically ignored by XAML readers.  
+ The XML white space concepts are not robust enough to apply to XAML, and are typically ignored by XAML readers.  
```

Comment:
white space concepts --> white-space concepts
also, unrelated to your PR, no comma after XAML